locales/fr.json

```diff
@@ -28,39 +28,39 @@
   "four_door": "Véhicule 4 portes",
   "compact": "Véhicule compact",
   "sedan": "Berline",
-  "suv": "VUS",
+  "suv": "SUV",
   "coupe": "Coupé",
-  "muscle": "Voiture de sport",
+  "muscle": "Grosse cylindrée",
   "sports_classic": "Voiture de sport classique",
   "sports": "Voiture de sport",
   "super": "Supercar",
-  "motorcycle": "Motocyclette",
+  "motorcycle": "Motos",
   "offroad": "Véhicule tout-terrain",
   "industrial": "Véhicule industriel",
   "utility": "Véhicule utilitaire",
-  "van": "Fourgon",
+  "van": "Van",
   "service": "Véhicule de service",
   "military": "Véhicule militaire",
   "truck": "Camion",
-  "north": "Direction nord",
-  "east": "Direction est",
-  "south": "Direction sud",
-  "west": "Direction ouest",
-  "male": "Mâle",
-  "female": "Femelle",
+  "north": "Direction Nord",
+  "east": "Direction Est",
+  "south": "Direction Sud",
+  "west": "Direction Ouest",
+  "male": "Homme",
+  "female": "Femme",
 
   "anon_call": "Appel anonyme entrant",
   "anon": "Anonyme",
   "hidden_number": "Numéro masqué",
   "call": "Appel entrant",
-  "vehicleshots": "Coups de feu tirés depuis un véhicule",
-  "shooting": "Décharge d'une arme à feu",
+  "vehicleshots": "Coups de feu depuis un véhicule",
+  "shooting": "Fusillade",
```

Comment:
Wouldn't "Décharge d'une arme à feu" make more sense? "Fusillade" is more like a series of gunshots...